lib/board.rb

```diff
@@ -48,10 +49,16 @@ def four_in_a_row_by_row(cells)
     end
 
     def diagonals(cells)
```

Comment:
This helps readability quite a bit. Was a little bit difficult to read before with the single line iterations.